Volkcor/Update internal repo version to update screenshots for Store PDP

volkcor opened the pull request:
Updating the internal repo version for updating the PDP screenshots in the Store to 0.0.101

mcooley commented:
@volkcor 0.0.101 is the latest internal package version which has picked up some loc changes. Let's use that one instead.

Also, can you update the PR description?

volkcor commented:
> @volkcor 0.0.101 is the latest internal package version which has picked up some loc changes. Let's use that one instead.
> 
> Also, can you update the PR description?

Updated version to 0.0.101 and added to PR description.
